audiobookshelf/docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: "3.7"
+volumes:
+  audiobooks:
+  podcasts:
+  config:
+  metadata:
+    
+services:
+  audiobookshelf:
+    image: ghcr.io/advplyr/audiobookshelf:latest
+    ports:
+      - 13378:80
```

Comment:
never specify a port when you are using traefik. Traefik sets the port, not the service being proxied.